packages/core/helper-plugin/lib/src/components/RelationInput/RelationInput.js

```diff
@@ -173,6 +198,10 @@ RelationInput.propTypes = {
   label: PropTypes.string.isRequired,
   labelLoadMore: PropTypes.string.isRequired,
   listHeight: PropTypes.string,
+  loadingMessage: PropTypes.shape({
+    id: PropTypes.string,
+    defaultMessage: PropTypes.string,
+  }).isRequired,
```

Comment:
I'd prefer to pass you `loadingMessage` as `PropTypes.string`, because then you never need to call `formatMessage` as part of the component. In case we move it into the DS, we can only have strings :)